tests/comfy_cli/jobs/test_jobs.py

```
@@ -884,6 +884,65 @@ def test_orphaned_flag_filters_to_watcher_crashed(monkeypatch):
+def test_reap_finalizes_nonterminal_record_with_dead_watcher_pid(monkeypatch):
+    """A `running` record carrying a dead pid + that pid's REAL create_time —
+    what a `comfy run --wait` killed from outside now leaves behind (BE-6641)
+    — is flipped by the reap to `error`/`watcher_crashed`, surfaces under
+    `--orphaned`, and the pid pair is cleared in the rewritten file."""
+    import psutil
+
+    from comfy_cli import jobs_state
+
+    # A real process that is provably gone: spawn, capture its create_time
+    # while alive, then terminate and reap it.
+    p = subprocess.Popen([sys.executable, "-c", "import time; time.sleep(30)"])
```

Pylint (4.0.6)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)